pkg/ctl/destroyContainer.go

```diff
@@ -0,0 +1,45 @@
+package ctl
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+// destroyContainerCmd represents the destroyContainer command
+var (
```

Comment:
we have to remove that, it's a daemon which expose endpoints, for executing commands we should have leappctl and UI